src/ledger/SyncingLedgerChain.h

```diff
@@ -5,6 +5,7 @@
 // of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
 
 #include <cstddef>
+#include <cstdint>
```

Comment:
out of curiosity, why do you need this include?

Reply:
I think `uint32_t` requires that.